frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -84,9 +84,12 @@ async function pollFunnel<T = FunnelStep[]>(apiParams: FunnelRequestParams): Pro
     return result
 }
 
-export const cleanFunnelParams = (filters: Partial<FilterType>): FilterType => {
+export const cleanFunnelParams = (filters: Partial<FilterType>, returnOnlyFunnelParams = false): FilterType => {
+    const breakdownEnabled =
+        filters.funnel_viz_type !== FunnelVizType.Trends && filters.funnel_viz_type !== FunnelVizType.TimeToConvert
+
     return {
-        ...filters,
+        ...(returnOnlyFunnelParams ? {} : filters),
```

Comment:
`returnOnlyFunnelParams` is not clear

Reply:
I renamed it to `discardFiltersNotUsedByFunnels` and added a comment to show its usage (to get funnel filters you can compare)